app/presenters/policy_area_content_item_presenter.rb

```diff
@@ -0,0 +1,94 @@
+class PolicyAreaContentItemPresenter
+
+  def initialize(policy_area)
+    @policy_area = policy_area
+  end
+
+  def exportable_attributes
+    {
+      "base_path" => base_path,
+      "format" => "policy_area",
+      "content_id" => content_id,
+      "title" => title,
+      "description" => description,
+      "public_updated_at" => public_updated_at,
+      "update_type" => "major",
+      "publishing_app" => "policy-publisher",
+      "rendering_app" => "finder-frontend",
+      "routes" => routes,
+      "details" => details,
+      "links" => {
+        "organisations" => organisations,
+        "topics" => [],
+        "related" => related,
+        "part_of" => part_of,
+      },
+    }
+  end
+
+private
+  attr_reader :policy_area
+
+  def base_path
+    "/government/policies/#{policy_area.slug}"
+  end
+
+  def content_id
+    policy_area.content_id
+  end
+
+  def title
+    policy_area.name
+  end
+
+  def description
+    ""
+  end
+
+  def public_updated_at
+    policy_area.updated_at
+  end
+
+  def routes
+    [
+      {
+        path: base_path,
+        type: "exact",
+      },
+      {
+        path: "#{base_path}.json",
+        type: "exact",
+      }
+    ]
+  end
+
+  def details
+    {
+      document_noun: "areas",
+      email_signup_enabled: false,
+      filter: {
+        policies: [policy_area.slug]
```

Comment:
I had a chat with @edds yesterday about this. We want to refer to them as `policies` when it comes to Rummager. He can elaborate.